internal/olm/client.go

```diff
@@ -164,6 +168,53 @@ func (c Client) UninstallVersion(ctx context.Context, version string) error {
 	return nil
 }
 
+func (c Client) GetInstalledVersion(ctx context.Context) (string, error) {
+	opts := client.InNamespace(olmNamespace)
+	csvs := &olmapiv1alpha1.ClusterServiceVersionList{}
+	if err := c.KubeClient.List(ctx, opts, csvs); err != nil {
+		if apierrors.IsNotFound(err) || meta.IsNoMatchError(err) {
+			return "", ErrOLMNotInstalled
+		}
+		return "", errors.Wrap(err, "failed to get OLM version from CSVs")
+	}
+	var pkgServerCSV *olmapiv1alpha1.ClusterServiceVersion
+	for _, csv := range csvs.Items {
+		if strings.HasPrefix(csv.GetName(), packageServerCSVNamePrefix) {
+			// There is more than one version of OLM installed in the cluster,
+			// so we can't resolve the version being used.
+			if pkgServerCSV != nil {
+				return "", errors.New("failed to get OLM version: more than one OLM version installed")
+			}
+			pkgServerCSV = &csv
+		}
+	}
+	if pkgServerCSV == nil {
+		return "", ErrOLMNotInstalled
+	}
+	return getOLMVersionFromPackageServerCSV(pkgServerCSV), nil
+}
+
+const (
+	packageServerCSVNamePrefix   = "packageserver."
+	packageServerOLMVersionLabel = "olm.version"
+)
+
+func getOLMVersionFromPackageServerCSV(csv *olmapiv1alpha1.ClusterServiceVersion) string {
+	// Package server CSV's from OLM versions > 0.10.1 have a label containing
```

Comment:
@joelanford the way OLM stores version information [changed](https://github.com/operator-framework/operator-lifecycle-manager/issues/935#issuecomment-513856066) in OLM version `> 0.10.1`. Trying `labels['olm.version']` first, then falling back to package server CSV name if that doesn't exist (for OLM versions `<= 0.10.1`), should work.